docs/content/reference/migration/migration-0-17.md

```diff
@@ -53,7 +53,7 @@ Now the override can be specified from the blueprint, removing the need to inclu
 
 ```python
 for t in range(1000):
-    rr.set_index("frame_nr", sequence=t)
+    rr.set_index("frame_nr", seq=t)
```

Comment:
I should have pointed that out in the other PR already, but the 0.17 migration guide should best reflect the 0.17 era situation 😬

Reply:
indeed - this was a search-and-replace mistake